src/middleware/open-graph/fonts.js

```diff
@@ -0,0 +1,125 @@
+const opentype = require('opentype.js');
+
+module.exports = class FontsProcessor {
+	constructor () {
+		/**
+		 * @type {Map<string, any>}
+		 * @private
+		 */
+		this._fonts = new Map();
+	}
+
+	_getFont (fontFamily) {
+		if (!this._fonts.has(fontFamily)) {
+			throw new Error('font is not loaded');
+		}
+
+		return this._fonts.get(fontFamily);
+	}
+
+	addFont (alias, fontPath) {
+		// todo: don't forget about loadSync here
```

Comment:
So what to do with this? I guess sync loading is better here, maybe rename the method to `addFontSync()` and remove comment?

Reply:
fixed